src/components/PricingCard/PricingCard.react.js

```diff
@@ -0,0 +1,36 @@
+//@flow
+
+import * as React from "react";
+
+import PricingCardCategory from "./PricingCardCategory.react";
+import PricingCardPrice from "./PricingCardPrice.react";
+import PricingCardAttributeList from "./PricingCardAttributeList.react";
+import PricingCardAttributeItem from "./PricingCardAttributeItem.react";
+import PricingCardButton from "./PricingCardButton.react";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +active?: boolean,
+  +title?: string | React.Node,
+  +body?: React.Node,
+|};
+
+function PricingCard({
+  className,
+  children,
+  active = false,
+  title,
+  body,
+}: Props): React.Node {
+  // TODO: Add component logic.
```

Comment:
Lets either add the component logic, or remove this file from this PR and add it in a separate one

Reply:
Added component logic, some enhancements may be needed